scripts/extract/config_generator/config_generator.py

```diff
@@ -0,0 +1,78 @@
+import os
+import json
+
+
+class TritonConfigGenerator(object):
+    def __init__(self):
+        self.overall_template = \
+'''name: "{alias}"
+platform: "{platform}"
+max_batch_size: "{max_bs}"
+input [
+{all_inputs}
+]
+output [
+{all_outputs}
+]
+'''
+        self.xput_template = \
+'''  {{ 
+    name: "{xput_name}"
+    data_type: {dtype}
+    dims: {dims}
+  }}'''
+
+        # Referring: https://docs.nvidia.com/deeplearning/triton-inference-server/master-user-guide/docs/model_configuration.html#datatypes
+        # as well as https://github.com/oracle/graphpipe/blob/master/docs/guide/user-guide/spec.md
```

Comment:
Why is graphpipe referred here?

Reply:
We may not implement the logic here. In my opinion, we may generate a config in the init container, as before.

Reply:
Sounds a better choice. Instead of for Triton, maybe an extractor for model stored in TensorRT engine files will be more useful.

Reply:
Yeah, I think so. Keeping generation logic in an extraction job does not really work.

Reply:
Yes, I misunderstood the issue kleveross/ormb#92 since there is only model registry repo. I'll close this PR after completing the design doc.